model_analyzer/config/input/config_command_profile.py

```diff
@@ -957,6 +958,15 @@ def _add_run_search_configs(self):
                 description="Maximum number of trials to profile when using Optuna",
             )
         )
+        self._add_config(
+            ConfigField(
+                "optuna_early_exit_threshold",
+                flags=["--optuna_early_exit_threshold"],
+                field_type=ConfigPrimitive(int),
+                default_value=DEFAULT_OPTUNA_EARLY_EXIT_THRESHOLD,
+                description="Number of trials to attempt before triggering early exit when using Optuna",
```

Comment:
Is this true? Code implies it is a number of attempts in a row with no improvement?

Reply:
The code is correct. It's the number of trials without improvement (min. number of trials must still be met). I'll update the description.